pubspec.yaml

```diff
@@ -20,7 +20,7 @@ dependencies:
   memoize: ^2.0.0
   meta: ^1.1.6
   path: ^1.5.1
-  react: ^5.3.0
+  react: ^5.3.1
```

Comment:
Doesn't exist yet, depends on release of https://github.com/cleandart/react-dart/pull/261